# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/x11vnc/hotfix-fnocommon.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

unchanged:
--- x11vnc-0.9.16/src/util.c.orig	2019-01-05 14:22:11.000000000 +0100
+++ x11vnc-0.9.16/src/util.c	2024-02-12 21:29:15.724961668 +0100
@@ -47,6 +47,9 @@
 #ifdef LIBVNCSERVER_HAVE_LIBPTHREAD
 MUTEX(x11Mutex);
 MUTEX(scrollMutex);
+MUTEX(clientMutex);
+MUTEX(inputMutex);
+MUTEX(pointerMutex);
 #endif

 int nfix(int i, int n);
only in patch2:
unchanged:
--- x11vnc-0.9.16/src/util.h.orig	2019-01-05 14:22:11.000000000 +0100
+++ x11vnc-0.9.16/src/util.h	2024-02-12 21:29:15.724961668 +0100
@@ -102,9 +102,9 @@
 #ifdef LIBVNCSERVER_HAVE_LIBPTHREAD
 extern MUTEX(x11Mutex);
 extern MUTEX(scrollMutex);
-MUTEX(clientMutex);
-MUTEX(inputMutex);
-MUTEX(pointerMutex);
+extern MUTEX(clientMutex);
+extern MUTEX(inputMutex);
+extern MUTEX(pointerMutex);
 #endif

 #define X_INIT INIT_MUTEX(x11Mutex)
